src/main/java/org/shanerx/tradeshop/utils/Utils.java

```diff
@@ -229,8 +229,8 @@ public void failedSign(SignChangeEvent e, ShopType shop, Message msg) {
 	 * @param e   Event to reset the sign for
 	 * @param msg The enum constant representing the error message
 	 */
-	public void failedTrade(PlayerInteractEvent e, Message msg) {
-		e.getPlayer().sendMessage(colorize(Setting.MESSAGE_PREFIX.getString() + msg));
+	public void failedTrade(PlayerInteractEvent e, MessagesEnum msg) {
+		plugin.getMessages().sendMessage(msg, e.getPlayer());
```

Comment:
why not something like `msg.send(Player)` ? Looks simpler, colorization should happen in there too.

Reply:
This is part of the reason I didn't end up liking this implementation, because it pulled the message methods out of the enum and put them into the controller requiring extended calls like this. My newer implementation fixes this though.